examples/run_nutation.py

```diff
@@ -2,110 +2,119 @@
 from pyspecdata import *
 import os
 import SpinCore_pp
-import socket
-import sys
-import time
+import h5py
+from Instruments.XEPR_eth import xepr
+from SpinCore_pp.ppg import run_spin_echo
 from datetime import datetime
+
 fl = figlist_var()
-#{{{ Edit here to set the actual field
-set_field = False
-if set_field:
-    B0 = 3497 # Determine this from Field Sweep
-    thisB0 = xepr().set_field(B0)
```

Comment:
we don't want to give the user control of the field so set_field would always be true so I just unindent and make it so